python/tank/templatekey.py

```diff
@@ -13,6 +13,8 @@
 """
 
 import re
+import datetime as dt
```

Comment:
import datetime # for datetime.time, datetime.date
from datetime import datetime # to avoid datetime.datetime

Isn't the second line defining a datetime variable that's hiding the
datetime module you just exported one live above?



On Wed, Jul 1, 2015 at 8:04 AM, Manne Öhrström <notifications@github.com>
wrote:

> In python/tank/templatekey.py
> <https://github.com/shotgunsoftware/tk-core/pull/183#discussion_r33653159>
> :
>
> > @@ -13,6 +13,8 @@
> >  """
> >
> >  import re
> > +import datetime as dt
>
> I always found it realy confusing when libraries are renamed like this. I
> suggest we do
>
> import datetime # for datetime.time, datetime.date
> from datetime import datetime # to avoid datetime.datetime
>
> Curious to hear what you think about that approach.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shotgunsoftware/tk-core/pull/183/files#r33653159>.
>